packages/api/endpoints/collections.js

```diff
@@ -21,11 +20,6 @@ const {
   CollectionSearch,
 } = require('@cumulus/db');
 const CollectionConfigStore = require('@cumulus/collection-config-store');
-const { getEsClient, Search } = require('@cumulus/es-client/search');
-const {
-  indexCollection,
```

Comment:
I want to be clear that the intention is to leave this function (indexCollection) in the code base?
I see that it's used in the api to provide ES interfaces, is the plan to leave the api still able to interact with ES so people can manage their cluster even while we're not adding new data to it?